python/dgl/mock_sparse/cpp_interface.py

```diff
@@ -0,0 +1,63 @@
+""" DGL sparse library C++ binding"""
+import os
+import torch
+from typing import List, Optional, Tuple
+
+package_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
+so_path = os.path.join(
+    package_path, "../tensoradapter/pytorch/libdgl_sparse.so"
+)
```

Comment:
Use the `find_lib_path` utility in https://github.com/dmlc/dgl/blob/master/python/dgl/_ffi/libinfo.py is better.

Reply:
Reuse the same directory as TensorAdapter now.